liquidcore/home/templates/homepage.html

```diff
@@ -33,4 +33,43 @@
 </p>
 {% endif %}
 
+
+<h2>Discovered nodes</h2>
+
+{% if nodes %}
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th>Interface</th>
+        <th>Hostname</th>
+        <th>IP</th>
+        <th>Discovered</td>
+      </tr>
+    </thead>
+    <tbody>
+  {% for interface, interface_nodes in nodes.items %}
+    {% for name, node in interface_nodes.items %}
+      {% if interface == status.dns_interface or not status.dns_interface %}
+        <tr>
+            <td>{{interface}}</td>
```

Comment:
Let's add some whitespace, `<td>{{ interface }}</td>`, I'm pretty sure that's the standard.